apps/smart-contracts/core/test/unit/TokenSender.test.ts

```diff
@@ -63,169 +61,112 @@ describe('=> TokenSender', () => {
     it('sets output token from constructor', async () => {
       expect(await tokenSender.getOutputToken()).to.eq(outputToken.address)
     })
-    it('does not set price', async () => {
-      expect(await tokenSender.getPrice()).to.eq(ZERO_ADDRESS)
+    it('does not set price oracle', async () => {
```

Comment:
nit:
```suggestion
    it('doesn't set price oracle', async () => {
```
same elsewhere